packages/subproviders/test/unit/private_key_wallet_subprovider_test.ts

```diff
@@ -0,0 +1,181 @@
+import { JSONRPCResponsePayload } from '@0xproject/types';
+import * as chai from 'chai';
+import * as ethUtils from 'ethereumjs-util';
+import * as _ from 'lodash';
+import Web3ProviderEngine = require('web3-provider-engine');
+
+import { GanacheSubprovider, PrivateKeyWalletSubprovider } from '../../src/';
+import {
+    DoneCallback,
+    LedgerCommunicationClient,
+    LedgerSubproviderErrors,
+    WalletSubproviderErrors,
+} from '../../src/types';
+import { chaiSetup } from '../chai_setup';
+import { fixtureData } from '../utils/fixture_data';
+import { reportCallbackErrors } from '../utils/report_callback_errors';
+
+chaiSetup.configure();
+const expect = chai.expect;
+
+describe('PrivateKeyWalletSubprovider', () => {
+    let subprovider: PrivateKeyWalletSubprovider;
+    before(async () => {
+        subprovider = new PrivateKeyWalletSubprovider(fixtureData.TEST_ACCOUNT_PRIVATE_KEY);
+    });
+    describe('direct method calls', () => {
+        describe('success cases', () => {
+            it('returns the account', async () => {
+                const accounts = await subprovider.getAccountsAsync();
+                expect(accounts[0]).to.be.equal(fixtureData.TEST_RPC_ACCOUNT_0);
+                expect(accounts.length).to.be.equal(1);
+            });
+            it('signs a personal message', async () => {
+                const data = ethUtils.bufferToHex(ethUtils.toBuffer(fixtureData.PERSONAL_MESSAGE_STRING));
+                const ecSignatureHex = await subprovider.signPersonalMessageAsync(data);
+                expect(ecSignatureHex).to.be.equal(fixtureData.PERSONAL_MESSAGE_SIGNED_RESULT);
+            });
+            it('signs a transaction', async () => {
+                const tx = {
+                    nonce: '0x00',
+                    gasPrice: '0x0',
+                    gas: '0x2710',
+                    to: '0x0000000000000000000000000000000000000000',
+                    value: '0x00',
+                    chainId: 3,
+                    from: fixtureData.TEST_RPC_ACCOUNT_0,
+                };
+                const txHex = await subprovider.signTransactionAsync(tx);
+                expect(txHex).to.be.equal(
+                    '0xf85f808082271094000000000000000000000000000000000000000080802aa018894834d89899f71f6d8e74e6992fea34914c3b6d8090495f738086ca18f15da056e3333ec6c7465512a49558a84b56ec358718feaf0b162bda9aa6c40824ede4',
```

Comment:
Should these be in the fixture module as well?

Reply:
Yep, they were using different networks. Refactored everything to network 42. Ledger now shares the same fixture data as private key wallet!